### PR TITLE
added statannot recipe

### PR DESCRIPTION
**Pull request by Matt-BF**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/statannot/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+MIT License
```
Comment (dopplershift) [inline recipes/statannot/LICENSE.txt:1]: It looks like `LICENSE` is already included in the PyPI tarball, so you can remove this.

File: recipes/statannot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+test:
+  imports:
+    - statannot
+
+about:
+  home: "https://github.com/webermarcolivier/statannot"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (dopplershift) [inline recipes/statannot/meta.yaml:37]: Should update to `LICENSE` and use the one in the tarball.

File: recipes/statannot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    - statannot
+
+about:
+  home: "https://github.com/webermarcolivier/statannot"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt 
+  summary: "add statistical annotations on an existing boxplot/barplot generated by seaborn."
+  doc_url: 
+  dev_url:
```
Comment (dopplershift) [inline recipes/statannot/meta.yaml:40]: Can you either fill these out or remove them?

**Review by dopplershift — CHANGES_REQUESTED** [review]
Looks good overall, just a couple minor issues.

File: recipes/statannot/meta.yaml (hunk 2 above)
Comment (Matt-BF) [reply recipes/statannot/meta.yaml]: Like this?

Comment (dopplershift) [reply recipes/statannot/meta.yaml]: Yup!

**Review by dopplershift — APPROVED** [review]

**Comment by Matt-BF** [comment]
Great! Thanks for the help!

**Pull request merged**